ci/docker-rust-nightly/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN set -x \
  && rustc --version \
  && cargo --version \
  && cargo install grcov \
+ && cargo install cargo-hack \
```

Comment:
I think we have installed this one in the stable image 🤔 
https://github.com/solana-labs/solana/blob/2839d51d5ee1a5d379b00b89062c6ff142b163f0/ci/docker-rust/Dockerfile#L37

let me check

Reply:
Yes, it is in the stable `Dockerfile`:
```
❯ git grep -w hack
ci/docker-rust/Dockerfile:37: && cargo install cargo-hack \
[...]
```

Reply:
i just figured it out.  accidentally built stable from wrong shell which had 1.16 checked out :upside_down_face:

Reply:
fwiw, I'm building docker image with a new method for installing node